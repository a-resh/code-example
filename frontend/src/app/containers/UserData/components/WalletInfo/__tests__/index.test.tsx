import * as React from 'react';
import { render } from '@testing-library/react';

import { WalletInfo } from '../index';

describe('<WalletInfo  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(
      <WalletInfo
        user={{ publicAddress: '', balance: 0, inGame: 0, btcAddress: '' }}
        setBtcAddress={() => {}}
      />,
    );
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
