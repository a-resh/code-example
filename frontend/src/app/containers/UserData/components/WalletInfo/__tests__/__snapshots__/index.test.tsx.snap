// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WalletInfo  /> should match snapshot 1`] = `
.c11 {
  width: 10px;
  height: 10px;
  background: center no-repeat url('assets/images/connected.svg');
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c0 {
  color: white;
  width: 100%;
  max-width: 820px;
}

.c0 h1 {
  font-size: 50px;
  font-weight: 100;
  margin: 0 0 15px 0;
}

.c0 h3 {
  font-weight: 300;
  font-size: 20px;
  margin: 15px 0;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  background-color: #272e38;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  padding: 0 0 30px 30px;
  width: 41%;
}

.c3 h1 {
  margin: 0 5px 0 0;
  line-height: 18px;
  font-weight: bold;
}

.c3 p {
  font-size: 10px;
  line-height: 50px;
}

.c3 h2 {
  margin: 0 5px 0 0;
  font-weight: bold;
  font-size: 36px;
  line-height: 27px;
}

.c3 h3 {
  margin-bottom: 30px;
}

.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  padding: 0 0 30px 30px;
  width: 41%;
  width: 50%;
}

.c6 h1 {
  margin: 0 5px 0 0;
  line-height: 18px;
  font-weight: bold;
}

.c6 p {
  font-size: 10px;
  line-height: 50px;
}

.c6 h2 {
  margin: 0 5px 0 0;
  font-weight: bold;
  font-size: 36px;
  line-height: 27px;
}

.c6 h3 {
  margin-bottom: 30px;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  padding: 0 0 30px 30px;
  width: 41%;
  width: 50%;
}

.c7 h1 {
  margin: 0 5px 0 0;
  line-height: 18px;
  font-weight: bold;
}

.c7 p {
  font-size: 10px;
  line-height: 50px;
}

.c7 h2 {
  margin: 0 5px 0 0;
  font-weight: bold;
  font-size: 36px;
  line-height: 27px;
}

.c7 h3 {
  margin-bottom: 30px;
}

.c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 60px;
  color: black;
  background-color: white;
  width: 20%;
  cursor: pointer;
}

.c8 p {
  font-weight: bold;
  font-size: 15px;
}

.c9 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  background-color: #232830;
  width: 80%;
  height: 60px;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding-right: 30px;
}

.c9 label {
  margin: 10px;
}

.c9 p {
  border: solid 2px black;
  padding: 0 10px;
}

.c10 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  background-color: black;
  font-size: 14px;
  margin: 0;
  height: 28px;
  line-height: 28px;
  padding: 10px;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  width: 59%;
}

@media (max-width:450px) {
  .c11 {
    margin: 0;
  }
}

@media (max-width:450px) {
  .c0 h1 {
    font-size: 27px;
    margin: 25px 0;
  }
}

@media (max-width:800px) {
  .c3 {
    width: 50%;
    padding: 0 0 15px 15px;
  }

  .c3 h1 {
    font-size: 36px;
  }
}

@media (max-width:450px) {
  .c3 {
    padding: 15px 0 0 15px;
  }

  .c3 h3 {
    font-size: 12px;
    margin: 0;
  }

  .c3 h1 {
    font-size: 27px;
    line-height: 36px;
  }

  .c3 h2 {
    font-size: 27px;
    line-height: 36px;
  }

  .c3 p {
    font-size: 8px;
  }
}

@media (max-width:800px) {
  .c6 {
    width: 50%;
    padding: 0 0 15px 15px;
  }

  .c6 h1 {
    font-size: 36px;
  }
}

@media (max-width:450px) {
  .c6 {
    padding: 15px 0 0 15px;
  }

  .c6 h3 {
    font-size: 12px;
    margin: 0;
  }

  .c6 h1 {
    font-size: 27px;
    line-height: 36px;
  }

  .c6 h2 {
    font-size: 27px;
    line-height: 36px;
  }

  .c6 p {
    font-size: 8px;
  }
}

@media (max-width:800px) {
  .c6 {
    display: none;
  }
}

@media (max-width:450px) {
  .c6 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    width: 100%;
  }
}

@media (max-width:800px) {
  .c7 {
    width: 50%;
    padding: 0 0 15px 15px;
  }

  .c7 h1 {
    font-size: 36px;
  }
}

@media (max-width:450px) {
  .c7 {
    padding: 15px 0 0 15px;
  }

  .c7 h3 {
    font-size: 12px;
    margin: 0;
  }

  .c7 h1 {
    font-size: 27px;
    line-height: 36px;
  }

  .c7 h2 {
    font-size: 27px;
    line-height: 36px;
  }

  .c7 p {
    font-size: 8px;
  }
}

@media (max-width:800px) {
  .c7 {
    width: 100%;
  }
}

@media (max-width:800px) {
  .c8 p {
    font-size: 12px;
  }
}

@media (max-width:450px) {
  .c8 {
    display: none;
  }
}

@media (max-width:800px) {
  .c9 {
    padding-right: 10px;
  }

  .c9 small {
    display: none;
  }
}

@media (max-width:450px) {
  .c9 {
    background-color: #202020;
    width: 100%;
    height: 25px;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
  }

  .c9 p,
  .c9 label,
  .c9 small {
    font-size: 9px;
  }

  .c9 small {
    display: inline;
  }
}

@media (max-width:800px) {
  .c10 {
    display: none;
  }
}

@media (max-width:450px) {
  .c10 {
    background-color: inherit;
    height: 17px;
    font-size: 9px;
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    padding: 4px;
  }

  .c10 p {
    line-height: 7px;
    border: none;
    padding: 0 5px;
  }
}

@media (max-width:450px) {
  .c2 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    background-color: #232830;
  }
}

@media (max-width:800px) {
  .c5 {
    width: 50%;
  }
}

@media (max-width:450px) {
  .c5 {
    width: 100%;
  }
}

<div
  class="c0"
>
  <h1>
    Your Wallet
  </h1>
  <div
    class="blocks__Column-sarv4y-1 c1"
  >
    <div
      class="c2"
    >
      <div
        class="blocks__Column-sarv4y-1 c3"
      >
        <h3>
          Total balance
        </h3>
        <div
          class="c4"
        >
          <h1>
            0
          </h1>
          <p>
            TOTM
          </p>
        </div>
      </div>
      <div
        class="c5"
      >
        <div
          class="blocks__Column-sarv4y-1 c6"
        >
          <h3>
            Amount staked
          </h3>
          <div
            class="c4"
          >
            <h2>
              0
            </h2>
            <p>
              TOTM
            </p>
          </div>
        </div>
        <div
          class="blocks__Column-sarv4y-1 c7"
        >
          <h3>
            Available to stake
          </h3>
          <div
            class="c4"
          >
            <h2>
              0
            </h2>
            <p>
              TOTM
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c4"
    >
      <div
        class="blocks__Center-sarv4y-2 c8"
      >
        <p>
          Add TOTM
        </p>
      </div>
      <div
        class="c9"
      >
        <label>
          BTC 
          <small>
            Address
          </small>
        </label>
        <p>
          1345678912345678987
        </p>
        <div
          class="blocks__Center-sarv4y-2 c10"
        >
          <p>
            Connected
          </p>
          <div
            class="c11"
            height="10"
            width="10"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
